### ISSUE TITLE
No response to commands

### ISSUE THREAD

**Issue by Harrison006** [DiscordPhone/DiscordPhone#22]
Hey, so ive built and ran the container, seems in discord itself its not responding to commands, says bots online. freepbx says its registered but to no avail

**Comment by nicolaipre** [comment]
Hi.

Did you follow the setup instructions? You need a Discord Bot token set in the `.env` file. You can get the token by following this guide: https://github.com/reactiflux/discord-irc/wiki/Creating-a-discord-bot-&-getting-a-token

**Comment by Harrison006** [comment]
Yes i have, could you provide an example .env configuration file?. or i can supply my own and redact my passwords.
